### PR TITLE
Refactor sendMessage to include the optional argument passing provided by Google

### PR DESCRIPTION
**Pull request by askhogan**

### CODE REVIEW

File: lib/chrome/Runtime.js [hunk 1]
```diff
@@ -145,6 +147,25 @@ function Runtime(chrome) {
...
+    }
+
+    //extensionId is optional and google type checks to determine which argument to use
+    if(extensionId === Object(extensionId)){
+      message = extensionId;
+    }
+
+    //message is not optional and google will throw an error without it
+    if(!message){
+      throw new Error("Invalid arguments to sendMessage.");
```
Comment (sethmcl) [inline lib/chrome/Runtime.js]: Could you please use single quotes here to conform with the eslint rules configured for the project?

**Comment by sethmcl** [comment]
Looks good thanks for the PR! Please see the one minor comment about quoting style, and then we can merge it :)

**Comment by askhogan** [comment]
I saw you fixed styling.  Went back through to add in some test coverage and merged in your styling fixes.  All tests are passing.  Also added some comments to the readme.

**Comment by sethmcl** [comment]
Hi sorry for delay on this. I had to make some changes to get the tests to pass, and I've merged it as PR #5. Please send another PR if you see any errors in this merge. Thanks!

**Pull request closed without merging**